### ISSUE TITLE
Review Notifications modal limitations

### ISSUE THREAD

**Issue by lmb244** [lmco/eurekastreams#89]
Notifications modal only displays up to 50 notifications. User may see 317 unread notifications (i.e. may be coming back from vacation), but only 50 notifications are retained in the notifications modal.  Product owners need to figure out what a reasonable usability scenario would be in this case.  Another possible solution suggestion would be to include check boxes in the notifications pop-up (checked by default?) that says something like "show seen notifications" so that you can only view those you haven't seen.

**Comment by billwolckenlmco** [comment]
I'd like the ability to delete individual notifications.  Besides cleaning out things that are no longer valuable, older notifications would then come up into view.  (Deleting wouldn't be the whole solution to the issue, just a contributing item.)

**Comment by lmb244** [comment]
I agree.  I would also like the ability to delete notifications.   I like to "clean stuff up".     At this point, I do believe they eventually roll off the notifications list.
